.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 python:
   - 2.7
+  - 3.4
```

Comment:
https://docs.travis-ci.com/user/languages/python/#Choosing-Python-versions-to-test-against indicates that Travis supports 3.5 out the box, so it probably makes sense to add that to the matrix.